test/unit/math/mix/fun/beta2_test.cpp

```diff
@@ -0,0 +1,14 @@
+#include <test/unit/math/test_ad.hpp>
```

Comment:
Shouldn't this be in the existing `beta_test` file?

Reply:
This is a windows thing. The file becomes too big for mingw if we have one file of tests so we have to break it up into two separate test files